Cargo.toml

```
@@ -5,6 +5,7 @@ members = [
     "fungible-token",
     "gear-feeds-router",
     "ping",
+    "DAO",
```

I'd change the directory name to the lower case (*nix-style) as we've done it with `nft`.
```suggestion
    "dao",
```